src/clients/mocks/MockConfigStoreClient.ts

```diff
@@ -30,15 +31,27 @@ export class MockConfigStoreClient extends AcrossConfigStoreClient {
     configStoreVersion: number,
     chainId = 1,
     mockUpdate = false,
-    enabledChainIdsOverride?: number[]
+    availableChainIdsOverride?: number[]
   ) {
     super(logger, configStore, eventSearchConfig, configStoreVersion);
     this.eventManager = mockUpdate ? getEventManager(chainId, this.eventSignatures) : null;
-    if (enabledChainIdsOverride) {
-      this.updateGlobalConfig(GLOBAL_CONFIG_STORE_KEYS.CHAIN_ID_INDICES, JSON.stringify(enabledChainIdsOverride), 0);
+    if (availableChainIdsOverride !== undefined && this.eventManager) {
```

Comment:
Can we use isDefined here ?

Reply:
Totally https://github.com/across-protocol/sdk-v2/pull/350/commits/ffa9fd63aba3ea7473c79f1fd28356b909be117c